src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -283,7 +295,13 @@ export const recordReleaseWarning = (
 export const documentDescription = () => {
   return (
     <div>
-      <p>File upload guidelines:</p>
+      <p>
+        You can upload your document in a pdf, .jpeg, or .png file format. You’ll
```

Comment:
This isn't the complete set of accepted file formats, though I don't know if we want to surface all of the accepted formats in this content snippet.